data/json/furniture_and_terrain/terrain-flora.json

```
@@ -1121,6 +1131,16 @@
       "items": [ { "item": "stick_long", "count": [ 3, 10 ] }, { "item": "splinter", "count": [ 10, 25 ] } ]
     }
   },
+  {
+    "type": "terrain",
+    "id": "t_tree_deadpine_warped",
+    "name": "warped dead pine tree",
+    "copy-from": "t_tree_deadpine",
+    "looks_like": "t_tree_deadpine",
+    "symbol": "Y",
+    "color": "brown",
+    "description": "The branches of this dead pine have been twisted into a shape that remind you of Sierpinski triangles.  It is pretty clear that it died due to whatever caused this."
```

```suggestion
    "description": "The branches of this dead pine have been twisted into a shape that remind you of Sierpinski triangles.  It did not survive getting warped like this."
```
Same, I think the formulation could be a bit more straight forward